charts/portworx-enterprise/v1.0.0/Chart.yaml

```diff
@@ -1,4 +1,4 @@
-name: portworx
+name: portworx-enterprise
```

Comment:
the PR should not touch the 1.0.0 folder.

For px-enterprise, is it possible to the folder name as `charts/portworx` but just change the chart name to portworx-enterprise?